package.xml

```diff
@@ -14,20 +14,28 @@
   <url type="repository">https://github.com/ros-perception/pcl_conversions</url>
   <url type="bugtracker">https://github.com/ros-perception/pcl_conversions/issues</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <build_depend>cmake_modules</build_depend>
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>eigen</build_depend>
   <build_depend>libpcl-all-dev</build_depend>
-  <build_depend>pcl_msgs</build_depend>
-  <build_depend>roscpp</build_depend>
+  <!-- TODO(clalancette): we don't support pcl_msgs yet,
+       reenable this once we do
+       <build_depend>pcl_msgs</build_depend> -->
   <build_depend>sensor_msgs</build_depend>
   <build_depend>std_msgs</build_depend>
 
-  <run_depend>libpcl-all</run_depend>
-  <run_depend>libpcl-all-dev</run_depend>
-  <run_depend>pcl_msgs</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>sensor_msgs</run_depend>
-  <run_depend>std_msgs</run_depend>
+  <build_export_depend>builtin_interfaces</build_export_depend>
+  <build_export_depend>eigen</build_export_depend>
+  <build_export_depend>libpcl-all-dev</build_export_depend>
```

Comment:
So, I'm not clear about the exact semantics of exec_depend with a header-only library such as this one.  Technically, it has zero runtime requirements, since it is header-only.  But I did notice that the ROS1 version of this package has <build_depend>, so I'm not sure.  Any advice?